src/main/java/uk/gov/hmcts/reform/sandl.snlevents/model/response/SessionInfo.java

```diff
@@ -13,6 +15,8 @@
 
 @Data
 @AllArgsConstructor
+//not sure
```

Comment:
if we aren't changing the production code to use SessionInfo without a no-args constructor and it's not adding huge value, then no, don't add this.

Reply:
done